code/game/machinery/computer/labor.dm

```diff
@@ -0,0 +1,165 @@
+var/list/labor_console_categories = list(
+	"Command" = command_positions,
+	"Civilian" = civilian_positions,
+	"Security" = security_positions,
+	"Engineering" = engineering_positions,
+	"Medical" = medical_positions,
+	"Science" = science_positions,
+	"Cargo" = cargo_positions,
+	)
+
+/obj/machinery/computer/labor
+	name = "Labor Administration Console"
+	desc = "According to the manual, you need to take a six-week Labor Administration Associate Training Course before you're qualified to navigate this console's complex interface. Being a HoP is hard work."
```

Comment:
Usually as a matter of style, descriptions do not contain abbreviations (being a head of personnel is hard work)